lib/internal/Magento/Framework/Model/ResourceModel/AbstractResource.php

```diff
@@ -92,14 +93,15 @@ public function commit()
          */
```

Comment:
The logic from this method was moved to `\Magento\Framework\Model\ExecuteCommitCallbacks`. Here it is left only for compatibility. That plugin basically cleans the pool of callbacks here and no callback should be executed. Could you please mark this method as deprecated. 

Also in order for this fix to be complete please fix the same here as well `\Magento\Framework\Model\ExecuteCommitCallbacks`

Reply:
The plugin `\Magento\Framework\Model\ExecuteCommitCallbacks` has already this fix.

Reply:
Can confirm that:
https://github.com/magento/magento2/blob/419c4168d4396cac5d1d0a6d3d741eb71861a145/lib/internal/Magento/Framework/Model/ExecuteCommitCallbacks.php#L38-L52